spec/es_helper.rb

```diff
@@ -0,0 +1,43 @@
+module ESHelper
+  def self.get_host_port
```

Comment:
from what I can see, this is only ever called from integration tests by way of `ESHelper#get_client`. Is this needed for non-integration? Should it error instead of providing the localhost value that isn't tested?